src/realm/sync/noinst/client_impl_base.hpp

```diff
@@ -750,7 +751,7 @@ class ClientImpl::Session {
 
     /// Update internal client state when a flx subscription becomes complete outside
     /// of the normal sync process. This can happen during client reset.
-    void non_sync_flx_completion(int64_t version);
+    void on_sync_flx_completion(int64_t version);
```

Comment:
wow, what a typo!

Reply:
Judging by the comment it should be prefixed with `non`

Reply:
so `on_non_sync_flx_completion`? or just revert it...

Reply:
I think on_non_sync_flx_completion does sound better

Reply:
Since it's calling `on_flx_sync_version_complete()` on the wrapper, I updated the name to be the same.